pkg/controller/user-cluster-controller-manager/pause.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2021 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package userclustercontrollermanager
+
+import (
+	"context"
+	"fmt"
+
+	kubermaticv1 "k8c.io/kubermatic/v2/pkg/crd/kubermatic/v1"
+
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type IsPausedChecker func(context.Context) (bool, error)
+
+func NewClusterPausedChecker(seedClient client.Client, clusterName string) IsPausedChecker {
+	return func(ctx context.Context) (bool, error) {
+		cluster := &kubermaticv1.Cluster{}
+		if err := seedClient.Get(ctx, types.NamespacedName{Name: clusterName}, cluster); err != nil {
+			if kerrors.IsNotFound(err) {
+				return false, nil
+			}
+
+			return false, fmt.Errorf("failed to get cluster %q: %v", clusterName, err)
```

Comment:
Any reasons using `%v` not `%w`? 
```suggestion
			return false, fmt.Errorf("failed to get cluster %q: %w", clusterName, err)
```